bgpd/bgp_ecommunity.c

```diff
@@ -373,6 +389,7 @@ enum ecommunity_token {
 	ecommunity_token_rt,
 	ecommunity_token_nt,
 	ecommunity_token_soo,
+	ecommunity_token_col,
```

Comment:
Any reason to shorten it to `col` instead of `color`?

Reply:
tacken into account